.docker/api-gateway-nginx.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    server_name localhost;
+    listen 80;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt { access_log off; log_not_found off; }
+
+    location ~ ^/scholarly-articles($|/) {
+        rewrite ^/scholarly-articles/?(.*)? /$1 break;
+        proxy_pass http://scholarly-articles_web;
+        proxy_redirect / /scholarly-articles;
```

Comment:
Handles the `Location` header (and `Refresh`). The current plan is to use `Link` for pagination, means they'll have to always be relative (so will be something like `Link: <?token=abcde>; rel="next"`).

Reply:
Makes sense as a plan, nginx can probably do some header manipulation but inserting arbitrary values in the middle of an existing `Link` would likely be difficult.